src/com/hms_networks/americas/sc/thingworx/TWConnectorMain.java

```diff
@@ -208,19 +220,22 @@ public static void main(String[] args) {
         "Starting " + TWConnectorConsts.CONNECTOR_NAME + " " + TWConnectorConsts.CONNECTOR_VERSION);
 
     // Calculate local time offset and configure queue
+    String timeZoneName = "";
     try {
-      TWTimeOffsetCalculator.calculateTimeOffsetMilliseconds(
-          connectorConfig.getFtpUser(), connectorConfig.getFtpPassword());
-    } catch (JSONException e) {
+      timeZoneOffsetMilliseconds = SCTimeUtils.getTimeZoneOffsetMilliseconds();
+      timeZoneName = SCTimeUtils.getTimeZoneName();
+    } catch (Exception e) {
       Logger.LOG_WARN(
-          "An error occured retrieving FTP user credentials from the configuration file.");
+          "An error occurred while retrieving the Flexy's time zone information. UTC time (no"
+              + " offset) will be used!");
       Logger.LOG_EXCEPTION(e);
     }
-    final long calculatedTimeOffsetMilliseconds =
-        TWTimeOffsetCalculator.getTimeOffsetMilliseconds();
-    HistoricalDataQueueManager.setLocalTimeOffset(calculatedTimeOffsetMilliseconds);
+    HistoricalDataQueueManager.setLocalTimeOffset(timeZoneOffsetMilliseconds);
     Logger.LOG_DEBUG(
-        "The local time offset is " + calculatedTimeOffsetMilliseconds + " milliseconds.");
+        "The offset for the configured time zone ("
```

Comment:
Move Log to try block above. Should an exception occur, this log line would not make sense. 
Also, can we convert this log to minutes and not milliseconds. This would be more helpful for users.

Reply:
no longer applicable -- change removed